docs-chef-io/content/inspec/v5/install.md

```diff
@@ -0,0 +1,141 @@
++++
+title = "Install Chef InSpec 5"
+gh_repo = "inspec"
++++
+
+You can install Chef InSpec on MacOS, Windows, and Linux systems.
+
+## Install Chef InSpec 5
+
+You can download the latest Chef InSpec package relevant to your operating system
+from [our Downloads Page](https://www.chef.io/downloads).
+
+You can also install Chef InSpec using an installer, script, or package
+manager.
+
+### macOS
+
+#### Homebrew
+
+Chef InSpec is available as a standalone [Homebrew](https://brew.sh/) package.
+Run the following command in your terminal to install Chef InSpec:
+
+```bash
+brew install chef/chef/inspec
+```
+
+While this command is running, Homebrew may prompt you to enter your macOS user account and
+password to complete the installation.
+
+#### CLI
+
+You can install Chef InSpec using a curl script.
+
+```bash
+curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec
```

Comment:
My understanding is that the install.sh script doesn't work with the new download APIs yet. Is there an alternative to using this?